doc/source/rllib.rst

```diff
@@ -1,347 +1,73 @@
+RLlib is an open-source library for reinforcement learning that offers both a collection of reference algorithms, and scalable primitives for composing new ones.
 
-You can find the code for RLlib `here on GitHub <https://github.com/ray-project/ray/tree/master/python/ray/rllib>`__, and the paper `here <https://arxiv.org/abs/1712.09381>`__.
+.. image:: rllib-stack.svg
 
-RLlib's policy optimizers serve as the basis for RLlib's reference algorithms, which include:
-
-- Proximal Policy Optimization (`PPO <https://github.com/ray-project/ray/tree/master/python/ray/rllib/ppo>`__) which is a proximal variant of `TRPO <https://arxiv.org/abs/1502.05477>`__.
-
-- Policy Gradients (`PG <https://github.com/ray-project/ray/tree/master/python/ray/rllib/pg>`__).
-
-- Asynchronous Advantage Actor-Critic (`A3C <https://github.com/ray-project/ray/tree/master/python/ray/rllib/a3c>`__).
-
-- Deep Q Networks (`DQN <https://github.com/ray-project/ray/tree/master/python/ray/rllib/dqn>`__).
-
-- Deep Deterministic Policy Gradients (`DDPG <https://github.com/ray-project/ray/tree/master/python/ray/rllib/ddpg>`__).
-
-- Ape-X Distributed Prioritized Experience Replay, including both `DQN <https://github.com/ray-project/ray/blob/master/python/ray/rllib/dqn/apex.py>`__ and `DDPG <https://github.com/ray-project/ray/blob/master/python/ray/rllib/ddpg/apex.py>`__ variants.
-
-- Evolution Strategies (`ES <https://github.com/ray-project/ray/tree/master/python/ray/rllib/es>`__), as described in `this paper <https://arxiv.org/abs/1703.03864>`__.
-
-These algorithms can be run on any `OpenAI Gym MDP <https://github.com/openai/gym>`__,
-including custom ones written and registered by the user.
-
-.. note::
-
-    To use RLlib's policy optimizers outside of RLlib, see the `policy optimizers documentation <policy-optimizers.html>`__.
+Learn more about RLlib's design by reading the `ICML paper <https://arxiv.org/abs/1712.09381>`__.
 
 Installation
 ------------
 
-RLlib has extra dependencies on top of **ray**. First, you'll need into install either PyTorch or TensorFlow.
-For usage of PyTorch models, visit the `PyTorch website <http://pytorch.org/>`__
-for instructions on installing PyTorch.
+RLlib has extra dependencies on top of ``ray``. First, you'll need to install either `PyTorch <http://pytorch.org/>`__ or `TensorFlow <https://www.tensorflow.org>`__. Then, install the Ray RLlib module:
 
 .. code-block:: bash
 
   pip install tensorflow  # or tensorflow-gpu
-
-Then, install Ray with extra RLlib dependencies:
-
-.. code-block:: bash
-
-  pip install 'ray[rllib]'
+  pip install ray[rllib]
 
 You might also want to clone the Ray repo for convenient access to RLlib helper scripts:
 
 .. code-block:: bash
 
   git clone https://github.com/ray-project/ray
+  cd ray/python/ray/rllib
 
+Training APIs
```

Comment:
We should have a quick start on this page (see https://gym.openai.com/, https://github.com/uber/horovod, etc).

Reply:
And likely split into 3 use cases: running a pre-made algorithm (1 line)